packages/webapp-events/package.json

```diff
@@ -5,16 +5,17 @@
     "typescript": "3.9.2"
   },
   "files": [
-    "dist"
+    "src",
+    "!src/**/*.ts"
```

Comment:
Why are you filtering out `.ts` files from being published?

Reply:
Because I only want to ship the distributables to npmjs. Our current "dist" folder also does not include plain TS.